lib/AST/UnqualifiedLookup.cpp

```diff
@@ -282,6 +282,10 @@ void UnqualifiedLookupFactory::lookUpTopLevelNamesInModuleScopeContext(
     return;
   }
 
+  bool inInterface = false;
```

Comment:
Nit: You can use `DeclContext::isInSwiftinterface()` to simplify this code a bit